hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/response/s3/multipart/TestS3MultipartUploadCompleteResponseWithFSO.java

```diff
@@ -133,6 +133,91 @@ public void testAddDBToBatch() throws Exception {
             omMetadataManager.getDeletedTable()));
   }
 
+
+
+  @Test
+  // similar to testAddDBToBatch(), but omBucketInfo is null
+  public void testAddDBToBatchWithNullBucketInfo() throws Exception {
```

Comment:
Thanks @k5342 .  Minor tweak - Would be great to refactor to share common code between this test and `testAddDBToBatch`.  Also, I saw the CI had a problem with a unrelated test and reran it for a green workflow.